data/mapcss-mapping.csv

```diff
@@ -37,7 +54,7 @@ power|tower;10;
 landuse|forest;[landuse=forest],[natural=wood];;name;int_name;11;
 landuse|grass;12;
 highway|secondary;13;
-natural|wood;14;landuse|forest
+deprecated:natural|wood;14;landuse|forest
```

Comment:
Don't know why you added "deprecated:" here, but actually, this entry defines landuse|forest.

Reply:
It doesn't define anything, its a deprecated type. The [natural=wood] is matched directly in the `landuse|forest` line just above.
I added `deprecated:` so that its actually visible that its deprecated and could be re-used for something else..

Reply:
This entry should **not** be reused.

Reply:
never? why? (I can add a comment with an explanation)

Reply:
because it is already landuse-forest. and 14 index is used for it

Reply:
mmm.. it was deprecated and replaced with `landuse-forest`
this means that the generator ignores this line completely
but the app, when it loads data/types.txt, replaces all id=14 in old mwms to id=11 (of the `landuse-forest`)
so this line `natural|wood;14;landuse|forest` is needed only to continue showing forest tagged with natural=wood for old mwms that were generated before the deprecation change
the change was done Jan 2020, there were no Organic Maps yet :) so its now safe to be re-used
what's wrong in this logic?

Reply:
Yes, landuse|forest is not a good example.